apps/air-discount-scheme/backend/migrations/20221114101732-comments-table.js

```diff
@@ -0,0 +1,30 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface) => {
+    return queryInterface.sequelize.query(`
+      BEGIN;
+        CREATE TABLE explicit_code (
+          id UUID NOT NULL,
+          code VARCHAR NOT NULL,
```

Comment:
There is no reason to store code.
Codes are temporary, why make them permanent?